src/Action/GetShortObjectDescriptionAction.php

```diff
@@ -64,14 +64,9 @@ public function __invoke(Request $request): Response
             $admin->setUniqid($uniqid);
         }
 
-        if (!$objectId) {
-            $objectId = null;
-        }
-
         $object = $admin->getObject($objectId);
-
-        if (!$object && 'html' === $request->get('_format')) {
-            return new Response();
+        if (!$object) {
+            throw new NotFoundHttpException(sprintf('Could not find subject for id "%s"', $objectId));
```

Comment:
This is changing the behavior, have you tried how it works before vs after?

Reply:
No, currently I can't try with master branch.
 
But If you look at the usage, it's only used in `edit_many_to_one` and `edit_one_to_one`.
```
{{ render(path('sonata_admin_short_object_information', {
    'code':     sonata_admin.field_description.associationadmin.code,
    'objectId': sonata_admin.field_description.associationadmin.id(sonata_admin.value),
    'uniqid':   sonata_admin.field_description.associationadmin.uniqid,
    'linkParameters': sonata_admin.field_description.options.link_parameters
})) }}
```

It's seems really weird to have nothing displayed in this case.